lutris/installer/commands.py

```diff
@@ -542,7 +542,7 @@ def _killable_process(self, func, *args, **kwargs):
             logger.debug("Process %s returned: %s", func, result)
             return result
 
-    def _extract_gog_game(self, file_id):
+    def _extract_innosetup(self, file_id):
```

Comment:
You've renamed this, but there are still calls to ``_extract_gog_game``. That seems wrong.

Reply:
Good catch!

Reply:
Fixed!